docs/_index.md

```diff
@@ -46,13 +46,8 @@ The script can be executed directly (i.e. 'curl python') or downloaded and then
 (e.g. in a CI environment).
 
 {{% warning %}}
-The `get-poetry.py` installer has been deprecated and removed. If you installed
-Poetry using `get-poetry.py`, please uninstall (as documented in the relevant
-step below), and then follow these installation instructions.
-
-The previous `install-poetry.py` script as included in the Poetry repository
-is deprecated and frozen. Please migrate to the standalone version described
-above, as the in-tree version is [scheduled to be removed](https://github.com/python-poetry/poetry/issues/6676).
+The `install-poetry.py` installer has been deprecated and removed.
+Please migrate to the standalone version described above.
```

Comment:
```suggestion
The `install-poetry.py` installer has been deprecated and removed from the Poetry repository.
Please migrate from the in-tree version to the standalone version described above.
```

Reply:
> `install-poetry.py` is mentioned in two more places in `_index.md` (pre-release versions and CI recommendations) that need to be changed.

I have updated the pre-release part. As to CI recommendations, it states clearly to download `install-poetry.py` from the `install.python-poetry.org` repo, so I would leave it as it is unless you have some idea how to rephrase it.

Reply:
I did not read the details. Should be fine.